src/commands/synthetics/command.ts

```diff
@@ -114,53 +113,66 @@ export class RunTestCommand extends Command {
     triggers: Trigger,
     startTime: number
   ) {
-    // Sort tests to show success first then non blocking failures and finally blocking failures.
-    tests.sort(this.sortTestsByOutcome(results))
+    const getTest = (publicId: string) => tests.find((t) => t.public_id === publicId) as Test
+
+    const sortedPollResults = Object.entries(results)
```

Comment:
What do you think about having a new interface including `Test`, `PollResult`, `ExecutionRule`, and `ResultOutcome` that we would pass along everywhere, it might simplify things a bit overall and avoid recomputing things multiple times?

Reply:
I would love to have such interface, for example

```ts
interface ResultToRender {
  test: Test,
  result: PollResult,
  executionRule: ExecutionRule,
  resultOutcome: ResultOutcome
}
```

However, wouldn't we need to have a new method for the `MainReporter`? Something that takes a `ResultToRender` as one of its arguments... 🤔

Reply:
Yes either that or we deconstruct it because reporter calls, maybe not call it `ResultToRender` to leave `rendering` to the actual reporter though

Reply:
So the `renderResults` method of the command would take a "result package" as a parameter instead, and this "result package" would be built earlier in the process, not here in this method, right?

Because if we build this package only here, and deconstruct it to pass the arguments to the reporter methods a bit later, I don't think it will be very useful ^^